etc/adt/config.template

```diff
@@ -124,6 +124,8 @@ DEPLOYMENT_KEYCLOAK_MODE=${DEPLOYMENT_KEYCLOAK_MODE}
 DEPLOYMENT_MATRIX_ENABLED=${DEPLOYMENT_MATRIX_ENABLED}
 DEPLOYMENT_MATRIX_HTTP_PORT=${DEPLOYMENT_MATRIX_HTTP_PORT}
 DEPLOYMENT_MATRIX_HTTPS_PORT=${DEPLOYMENT_MATRIX_HTTPS_PORT}
+DEPLOYMENT_MATRIX_ADMIN_USERNAME=${DEPLOYMENT_MATRIX_ADMIN_USERNAME}
+DEPLOYMENT_MATRIX_ADMIN_PASSWORD=${DEPLOYMENT_MATRIX_ADMIN_PASSWORD}
```

Comment:
These two variables should be declared in the _functions.sh file to avoid unbound variable issues and automatically provided using INSTANCE_KEY and a hashed password. We cannot provide for each instance a username and password